src/components/registrationForm.js

```diff
@@ -0,0 +1,55 @@
+import locators from '../../locators/registration.js'
+import { expect } from '@playwright/test'; 
+
+export default class RegistrationForm {
+    constructor(page) {
+        this._page = page
+        this.registrationModalTitle = page.locator(locators.registrationModalTitle)
+        this.nameLabel = page.locator(locators.nameLabel)
+        this.lastNameLabel = page.locator(locators.lastNameLabel)
+        this.emailLabel = page.locator(locators.emailLabel)
+        this.passwordLabel = page.locator(locators.passwordLabel)
+        this.reenterPasswordLabel = page.locator(locators.reenterPasswordLabel)
+        this.registrationButton = page.locator(locators.registrationButton)
+        this.allertForField = page.locator(locators.allertForField)
+        this.nameField = page.locator(locators.name)
+        this.lastNameField = page.locator(locators.lastName)
+        this.emailField = page.locator(locators.email)
+        this.passwordField = page.locator(locators.password)
+        this.repeatePasswordField = page.locator(locators.repeatPassword)
+    }    
+
+    async checkTextElements (pageElement, expectedText) {
+        await expect.soft(pageElement).toHaveText(expectedText)
+    }
+
+    async isAllertForValidDataAbsent (locator, data) {
```

Comment:
you should not pass selectors to Page Objects

selectors should be encapsulated in Page Objects

Reply:
the same for other methods